BlueShift-iOS-SDK/BlueShiftDeviceData.m

```diff
@@ -43,7 +44,7 @@ - (NSString *)deviceUUID {
             if (bundleId != nil) {
                 deviceUUID = [NSString stringWithFormat:@"%@:%@", self.deviceIDFV,bundleId];
             } else {
-                NSLog(@"[BlueShift] - ERROR: Failed to get the bundle Id");
+                [BlueshiftLog logError:nil withDescription:@"Failed to get the bundle Id. Use different DeviceIdSource." methodName:nil];
```

Comment:
I think we shouldn't ask them to choose a different `DeviceIdSource`. Let them know we couldn't get the bundle id this time and that's it.

Reply:
Okay. I have altered the log message to "Failed to get the bundle Id."